dao-voting/SPECIFICATION.md

```diff
@@ -0,0 +1,114 @@
+Table of Content
+- [Actions that users are able to perform](#actions-that-users-are-able-to-perform)
+  - [Computation](#computation)
+    - [`constructor()`](#constructor)
+    - [`create_proposal()`](#create_proposal)
+    - [`deposit()`](#deposit)
+    - [`withdraw()`](#withdraw)
+    - [`vote()`](#vote)
+    - [`execute()`](#execute)
+    - [`unlock_votes()`](#unlock_votes)
+  - [State Check](#state-check)
+    - [`balance()`](#balance)
+    - [`user_balance()`](#user_balance)
+    - [`user_votes()`](#user_votes)
+    - [`proposal()`](#proposal)
+    - [`governance_token_id()`](#governance_token_id)
+    - [`proposal_count()`](#proposal_count)
+- [Sequence diagram](#sequence-diagram)
+
+# Overview
+
+This document provides an overview of the application.
+
+It outline the use cases, i.e. desirable functionality, in addition to requirements for the smart contract and the user interface.
```

Comment:
```suggestion
It outlines the use cases, i.e. desirable functionality, in addition to requirements for the smart contract and the user interface.
```

Reply:
oh no, I think every spec might have this issue